packages/grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -350,18 +356,19 @@ export const KeyboardNavigationMixin = (superClass) =>
           dy = -1;
           break;
         case 'PageDown':
-          dy = visibleItemsCount;
+          {
+            const currentRowIndex = this.__getIndexInGroup(activeRow, this._focusedItemIndex);
+            this._scrollToFlatIndex(currentRowIndex); // scroll the current row to the top...
```

Comment:
If the pagedown press occurs in the grid header, this will cause the body section to unexpectedly scroll:

https://github.com/vaadin/web-components/assets/1222264/f052673d-9f8d-47f2-bcdf-882330bad771

I'll add a test case and check that the current group is body before calling `_scrollToFlatIndex`